---
title: 'The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene
  structure, evolution, and expression.'
date: '2012-09-01'
draft: true
publishDate: '2020-11-15T23:59:22.085787Z'
authors:
- Thomas Derrien
- Rory Johnson
- Giovanni Bussotti
- Andrea Tanzer
- Sarah Djebali
- Hagen Tilgner
- Gregory Guernec
- David Martin
- Angelika Merkel
- David G Knowles
- Julien Lagarde
- Lavanya Veeravalli
- Xiaoan Ruan
- Yijun Ruan
- Timo Lassmann
- Piero Carninci
- James B Brown
- Leonard Lipovich
- Jose M Gonzalez
- Mark Thomas
- Carrie A Davis
- Ramin Shiekhattar
- Thomas R Gingeras
- Tim J Hubbard
- Cedric Notredame
- Jennifer Harrow
- Roderic Guigo
publication_types:
- '2'
abstract: The human genome contains many thousands of long noncoding RNAs (lncRNAs).
  While several studies have demonstrated compelling biological and disease roles
  for individual examples, analytical and experimental approaches to investigate these
  genes have been hampered by the lack of comprehensive lncRNA annotation. Here, we
  present and analyze the most complete human lncRNA annotation to date, produced
  by the GENCODE consortium within the framework of the ENCODE project and comprising
  9277 manually annotated genes producing 14,880 transcripts. Our analyses indicate
  that lncRNAs are generated through pathways similar to that of protein-coding genes,
  with similar histone-modification profiles, splicing signals, and exon/intron lengths.
  In contrast to protein-coding genes, however, lncRNAs display a striking bias toward
  two-exon transcripts, they are predominantly localized in the chromatin and nucleus,
  and a fraction appear to be preferentially processed into small RNAs. They are under
  stronger selective pressure than neutrally evolving sequences-particularly in their
  promoter regions, which display levels of selection comparable to protein-coding
  genes. Importantly, about one-third seem to have arisen within the primate lineage.
  Comprehensive analysis of their expression in multiple human organs and brain regions
  shows that lncRNAs are generally lower expressed than protein-coding genes, and
  display more tissue-specific expression patterns, with a large fraction of tissue-specific
  lncRNAs expressed in the brain. Expression correlation analysis indicates that lncRNAs
  show particularly striking positive correlation with the expression of antisense
  coding genes. This GENCODE annotation represents a valuable resource for future
  studies of lncRNAs.
featured: false
publication: '*Genome research*'
tags:
- '"Alternative Splicing"'
- '"Animals"'
- '"Cell Nucleus"'
- '"Cluster Analysis"'
- '"Databases"'
- '"Evolution"'
- '"Exons"'
- '"Gene Expression Profiling"'
- '"Gene Expression Regulation"'
- '"Genetic"'
- '"genetics"'
- '"Histones"'
- '"Humans"'
- '"Long Noncoding"'
- '"Messenger"'
- '"metabolism"'
- '"Molecular"'
- '"Molecular Sequence Annotation"'
- '"Open Reading Frames"'
- '"Organ Specificity"'
- '"Post-Transcriptional"'
- '"Primates"'
- '"RNA"'
- '"RNA Processing"'
- '"RNA Splice Sites"'
- '"Selection"'
- '"Transcription"'
doi: 10.1101/gr.132159.111
---

