---
title: A comparative genome approach to marker ordering.
date: '2007-01-01'
draft: true
publishDate: '2021-06-30T13:50:34.461872Z'
authors:
- T Faraut
- S de Givry
- P Chabrier
- T Derrien
- F Galibert
- C Hitte
- T Schiex
publication_types:
- '2'
abstract: 'MOTIVATION: Genome maps are fundamental to the study of an organism and
  essential in the process of genome sequencing which in turn provides the ultimate
  map of the genome. The increased number of genomes being sequenced offers new opportunities
  for the mapping of closely related organisms. We propose here an algorithmic formalization
  of a genome comparison approach to marker ordering. RESULTS: In order to integrate
  a comparative mapping approach in the algorithmic process of map construction and
  selection, we propose to extend the usual statistical model describing the experimental
  data, here radiation hybrids (RH) data, in a statistical framework that models additionally
  the evolutionary relationships between a proposed map and a reference map: an existing
  map of the corresponding orthologous genes or markers in a closely related organism.
  This has concretely the effect of exploiting, in the process of map selection, the
  information of marker adjacencies in the related genome when the information provided
  by the experimental data is not conclusive for the purpose of ordering. In order
  to compute efficiently the map, we proceed to a reduction of the maximum likelihood
  estimation to the Traveling Salesman Problem. Experiments on simulated RH datasets
  as well as on a real RH dataset from the canine RH project show that maps produced
  using the likelihood defined by the new model are significantly better than maps
  built using the traditional RH model. AVAILABILITY: The comparative mapping approach
  is available in the last version of de Givry,S. et al. [(2004) Bioinformatics, 21,
  1703-1704, www.inra.fr/mia/T/CarthaGene], a free (the LKH part is free for academic
  use only) mapping software in C++, including LKH (Helsgaun,K. (2000) Eur. J. Oper.
  Res., 126, 106-130, www.dat.ruc.dk/keld/research/LKH) for maximum likelihood computation.'
featured: false
publication: '*Bioinformatics (Oxford, England)*'
tags:
- '"DNA"'
- '"Algorithms"'
- '"Chromosome Mapping"'
- '"Genetic"'
- '"methods"'
- '"Sequence Analysis"'
- '"Software"'
- '"genetics"'
- '"Species Specificity"'
- '"Evolution"'
- '"Molecular"'
- '"Computer Simulation"'
- '"Genetic Markers"'
- '"Linkage Disequilibrium"'
- '"Models"'
- '"Nucleic Acid"'
- '"Sequence Homology"'
- '"Statistical"'
url_pdf: www.dat.ruc.dk/keld/research/ http://www.ncbi.nlm.nih.gov/pubmed/17237105
doi: 10.1093/bioinformatics/btl321
---

