---
title: Transcriptome profiling of mouse samples using nanopore sequencing of cDNA
  and RNA molecules.
date: '2019-10-01'
draft: false
publishDate: '2020-11-16T09:04:01.143794Z'
authors:
- Camille Sessegolo
- Corinne Cruaud
- Corinne Da Silva
- Audric Cologne
- Marion Dubarry
- Thomas Derrien
- Vincent Lacroix
- Jean-Marc Aury
publication_types:
- '2'
abstract: Our vision of DNA transcription and splicing has changed dramatically with
  the introduction of short-read sequencing. These high-throughput sequencing technologies
  promised to unravel the complexity of any transcriptome. Generally gene expression
  levels are well-captured using these technologies, but there are still remaining
  caveats due to the limited read length and the fact that RNA molecules had to be
  reverse transcribed before sequencing. Oxford Nanopore Technologies has recently
  launched a portable sequencer which offers the possibility of sequencing long reads
  and most importantly RNA molecules. Here we generated a full mouse transcriptome
  from brain and liver using the Oxford Nanopore device. As a comparison, we sequenced
  RNA (RNA-Seq) and cDNA (cDNA-Seq) molecules using both long and short reads technologies
  and tested the TeloPrime preparation kit, dedicated to the enrichment of full-length
  transcripts. Using spike-in data, we confirmed that expression levels are efficiently
  captured by cDNA-Seq using short reads. More importantly, Oxford Nanopore RNA-Seq
  tends to be more efficient, while cDNA-Seq appears to be more biased. We further
  show that the cDNA library preparation of the Nanopore protocol induces read truncation
  for transcripts containing internal runs of T's. This bias is marked for runs of
  at least 15 T's, but is already detectable for runs of at least 9 T's and therefore
  concerns more than 20% of expressed transcripts in mouse brain and liver. Finally,
  we outline that bioinformatics challenges remain ahead for quantifying at the transcript
  level, especially when reads are not full-length. Accurate quantification of repeat-associated
  genes such as processed pseudogenes also remains difficult, and we show that current
  mapping protocols which map reads to the genome largely over-estimate their expression,
  at the expense of their parent gene.
featured: false
publication: '*Scientific reports*'
doi: 10.1038/s41598-019-51470-9
---

