---
title: Assessment of transcript reconstruction methods for RNA-seq
date: '2013-01-01'
draft: true
publishDate: '2021-06-30T13:50:35.136024Z'
authors:
- Tamara Steijger
- Josep F. Abril
- Pär G. Engström
- Felix Kokocinski
- Martin Akerman
- Tyler Alioto
- Giovanna Ambrosini
- Stylianos E. Antonarakis
- Jonas Behr
- Paul Bertone
- Regina Bohnert
- Philipp Bucher
- Nicole Cloonan
- Thomas Derrien
- Sarah Djebali
- Jiang Du
- Sandrine Dudoit
- Mark Gerstein
- Thomas R. Gingeras
- David Gonzalez
- Sean M. Grimmond
- Roderic Guigó
- Lukas Habegger
- Jennifer Harrow
- Tim J. Hubbard
- Christian Iseli
- Géraldine Jean
- André Kahles
- Julien Lagarde
- Jing Leng
- Gregory Lefebvre
- Suzanna Lewis
- Ali Mortazavi
- Peter Niermann
- Gunnar Rätsch
- Alexandre Reymond
- Paolo Ribeca
- Hugues Richard
- Jacques Rougemont
- Joel Rozowsky
- Michael Sammeth
- Andrea Sboner
- Marcel H. Schulz
- Steven M.J. Searle
- Naryttza Diaz Solorzano
- Victor Solovyev
- Mario Stanke
- Brian J. Stevenson
- Heinz Stockinger
- Armand Valsesia
- David Weese
- Simon White
- Barbara J. Wold
- Jie Wu
- Thomas D. Wu
- Georg Zeller
- Daniel Zerbino
- Michael Q. Zhang
publication_types:
- '2'
abstract: We evaluated 25 protocol variants of 14 independent computational methods
  for exon identification, transcript reconstruction and expression-level quantification
  from RNA-seq data. Our results show that most algorithms are able to identify discrete
  transcript components with high success rates but that assembly of complete isoform
  structures poses a major challenge even when all constituent elements are identified.
  Expression-level estimates also varied widely across methods, even when based on
  similar transcript models. Consequently, the complexity of higher eukaryotic genomes
  imposes severe limitations on transcript recall and splice product discrimination
  that are likely to remain limiting factors for the analysis of current-generation
  RNA-seq data.
featured: false
publication: '*Nature Methods*'
doi: 10.1038/nmeth.2714
---

