@article{Derrien2012, abstract = {We present a fast mapping-based
algorithm to compute the mappability of each region of a reference
genome up to a specified number of mismatches. Knowing the mappability
of a genome is crucial for the interpretation of massively parallel
sequencing experiments. We investigate the properties of the mappability
of eukaryotic DNA/RNA both as a whole and at the level of the gene
family, providing for various organisms tracks which allow the
mappability information to be visually explored. In addition, we show
that mappability varies greatly between species and gene classes.
Finally, we suggest several practical applications where mappability can
be used to refine the analysis of high-throughput sequencing data (SNP
calling, gene expression quantification and paired-end experiments).
This work highlights mappability as an important concept which deserves
to be taken into full account, in particular when massively parallel
sequencing technologies are employed. The GEM mappability program
belongs to the GEM (GEnome Multitool) suite of programs, which can be
freely downloaded for any use from its website
(http://gemlibrary.sourceforge.net).}, author = {Derrien, Thomas and
Estelle, Jordi and Marco Sola, Santiago and Knowles, David G and
Raineri, Emanuele and Guigo, Roderic and Ribeca, Paolo}, doi =
{10.1371/journal.pone.0030377}, issn = {1932-6203 (Electronic)}, journal
= {PloS one}, keywords = {Algorithms,Computational Biology,Genome,
Human,High-Throughput Nucleotide Sequencing,Humans,Sequence Analysis,
DNA,genetics,methods}, language = {eng}, number = {1}, pages = {e30377},
pmid = {22276185}, title = {Fast computation and applications of genome
mappability.}, volume = {7}, year = {2012} }
